src/composables/useHotKey/index.ts

```diff
@@ -57,15 +91,20 @@ const eventHandler = (callback, options) => (event) => {
 		event.stopPropagation()
 	}
 	callback(event)
+	}
 }
 
 /**
- * @param {true|string|string[]|Function} keysOrFilter - keyboard key or keys to listen to, or filter function
- * @param {Function} callback - callback function
- * @param {object} options - composable options
+ * @param keysOrFilter - keyboard key or keys to listen to, or filter function
+ * @param callback - callback function
+ * @param options - composable options
  * @see docs/composables/usekeystroke.md
  */
-export function useHotKey(keysOrFilter, callback = () => {}, options = {}) {
+export function useHotKey(
+	keysOrFilter: string | string[] | ((e: KeyboardEvent) => boolean),
```

Comment:
do not skip `true`, it's a specific value to listen to all kbd events
```suggestion
	keysOrFilter: true | string | string[] | ((e: KeyboardEvent) => boolean),
```